src/misc/lv_obj_property.c

```diff
@@ -0,0 +1,179 @@
+/**
+ * @file lv_obj_id.c
+ *
+ */
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "../core/lv_obj.h"
+#include "../stdlib/lv_string.h"
+#include "lv_obj_property.h"
+
+#if LV_USE_OBJ_PROPERTY
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+typedef void (*lv_property_set_int_t)(struct _lv_obj_t *, int32_t);
+typedef void (*lv_property_set_pointer_t)(struct _lv_obj_t *, const void *);
+typedef void (*lv_property_set_color_t)(struct _lv_obj_t *, lv_color_t);
+typedef lv_result_t (*lv_property_setter_t)(struct _lv_obj_t *, lv_prop_id_t, const lv_property_t *);
+
+typedef int32_t (*lv_property_get_int_t)(const struct _lv_obj_t *);
+typedef void * (*lv_property_get_pointer_t)(const struct _lv_obj_t *);
+typedef lv_color_t (*lv_property_get_color_t)(const struct _lv_obj_t *);
+typedef lv_result_t (*lv_property_getter_t)(const struct _lv_obj_t *, lv_prop_id_t, lv_property_t *);
+
+/**********************
+ *  STATIC PROTOTYPES
+ **********************/
+
+static lv_result_t obj_property(lv_obj_t * obj, lv_prop_id_t id, lv_property_t * value, bool set);
+
+/**********************
+ *  STATIC VARIABLES
+ **********************/
+
+/**********************
+ *      MACROS
+ **********************/
+
+/**********************
+ *   GLOBAL FUNCTIONS
+ **********************/
+
+lv_result_t lv_obj_property_set(lv_obj_t * obj, const lv_property_t * value)
```

Comment:
I wonder if we can have a function later like:
```c
#define lv_obj_set(obj, prop, v)    lv_obj_propery_set_direct(obj, prop, (lv_value_group_t) v);
```
(The names are just examples)

Reply:
We can simply rename `lv_obj_set_property` to `lv_obj_set`? 
Let's firstly see how `lv_obj_set_property` can be used. If it can be widely used, then I suppose it's better to add `lv_obj_set`.

Reply:
Sounds good to me. We can easily refactor it at any time.